### PR TITLE
feat(specs): add more descriptions to the Ingestion API

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-893

### Changes included:

This PR adds some description to the Ingestion API.

### CODE REVIEW

File: specs/ingestion/paths/runs/events/events.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  summary: Get a list of events for a specific run.
-  description: Get a list of events for a specific run.
+  summary: Get a list of events.
+  description: Get a list of events associated to the given runID, for the given query parameters, with pagination details.
   operationId: getEvents
   parameters:
-    - $ref: '../../../common/parameters.yml#/runID'
+    - $ref: '../../../common/parameters.yml#/pathRunID'
     - $ref: '../../../common/parameters.yml#/itemsPerPage'
     - $ref: '../../../common/parameters.yml#/page'
```
Comment (millotp) [inline specs/ingestion/paths/runs/events/events.yml:10]: I though I removed those in #1277

File: specs/ingestion/paths/runs/tasks/taskID.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+get:
```
Comment (millotp) [inline specs/ingestion/paths/runs/tasks/taskID.yml:1]: same, this should be gone

File: specs/ingestion/spec.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.2
 info:
   title: Ingestion API
-  description: API powering the Data Ingestion feature of Algolia.
+  description: API powering the Data Ingestion connectors of Algolia.
```
Comment (millotp) [inline specs/ingestion/spec.yml:4]: we also support the Flow API, even though it's not spec here

File: specs/ingestion/spec.yml [hunk 4]
```diff
@@ -22,45 +22,41 @@ security:
...
 paths:
-  # authentications api
   /1/authentications:
     $ref: 'paths/authentications/authentications.yml'
   /1/authentications/search:
     $ref: 'paths/authentications/searchAuthentications.yml'
   /1/authentications/{authenticationID}:
     $ref: 'paths/authentications/authenticationID.yml'
 
-  # destinations api
```
Comment (millotp) [inline specs/ingestion/spec.yml:47]: why remove those ? we have the same in other specs

**Review by millotp — COMMENTED** [review]
Huge work thank you !

File: specs/ingestion/spec.yml (hunk 4 above)
Comment (shortcuts) [reply specs/ingestion/spec.yml:47]: thought it was redundant considering how well named the ingestion API is. can revert if you wish

Comment (millotp) [reply specs/ingestion/spec.yml:47]: I don't really care it was for consistency

File: specs/ingestion/spec.yml (hunk 3 above)
Comment (shortcuts) [reply specs/ingestion/spec.yml:4]: yup true, but flow is fully internal right? not sure we want to mention it for now

Comment (millotp) [reply specs/ingestion/spec.yml:4]: I still don't know ahah

Comment (shortcuts) [reply specs/ingestion/spec.yml:4]: XD true I guess that answers the subject, let's leave it like that it's a bit blurry still

File: specs/ingestion/spec.yml (hunk 4 above)
Comment (shortcuts) [reply specs/ingestion/spec.yml:47]: done!

**Comment by shortcuts** [comment]
> nice !

thanks for helping!

**Pull request merged**